<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RentDialog</class>
 <widget class="QDialog" name="RentDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>507</width>
    <height>378</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Rent out items</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="input_hirer_pushButton">
       <property name="text">
        <string>Set &amp;hirer</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="input_duration_pushButton">
       <property name="text">
        <string>Change &amp;duration</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>&amp;Item</string>
         </property>
         <property name="buddy">
          <cstring>input_item_lineEdit</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="input_item_lineEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="input_addItem_pushButton">
       <property name="text">
        <string>&amp;Add item</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="input_rentalgroup_pushButton">
       <property name="text">
        <string>Change &amp;rentalgroup</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="input_finish_pushButton">
       <property name="text">
        <string>&amp;Finish</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="input_editNote_pushButton">
       <property name="text">
        <string>Edit &amp;note</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="input_park_pushButton">
       <property name="text">
        <string>&amp;Park</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="input_cancel_pushButton">
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QTextBrowser" name="output_contract_textBrowser"/>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>input_hirer_pushButton</tabstop>
  <tabstop>input_duration_pushButton</tabstop>
  <tabstop>input_item_lineEdit</tabstop>
  <tabstop>input_addItem_pushButton</tabstop>
  <tabstop>input_rentalgroup_pushButton</tabstop>
  <tabstop>input_finish_pushButton</tabstop>
  <tabstop>input_cancel_pushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
